Ansible/ansible_collections/jfrog/platform/roles/postgres/tasks/install.yml

```diff
@@ -31,6 +31,15 @@
   environment:
     LC_ALL: "{{ postgres_locale }}"
 
+- name: Grab the FIPS status
+  command: cat /proc/sys/crypto/fips_enabled
+  register: _fips_enabled
+  ignore_errors: true
+
+- name: Store FIPS status for use
```

Comment:
There is no need to use `set_fact`. So you can remove this task

Reply:
Agreed, this could be removed.

Reply:
Agreed, this could be removed. My original plan was to not have a default variable backing this task but that has changed due to the previous request so this can be removed.